blocked-edges/4.12.48-NetPolicyTimeoutsHostNetworkedPodTraffic.yaml

```diff
@@ -7,9 +7,17 @@ message: |-
   Clusters where Ingress Controller uses HostNetwork endpoint publishing strategy and certain NetworkPolicy configurations
   may experience network traffic disruptions in namespaces that allowed traffic before the upgrade.
 matchingRules:
-- type: PromQL
-  promql:
-    promql: |
-      group(max_over_time(apiserver_storage_objects{resource="networkpolicies.networking.k8s.io"}[1h]) > 0)
-      or
-      0 * group(max_over_time(apiserver_storage_objects[1h]))
+  - type: PromQL
```

Comment:
Apologize for the whitespace - it is coming from an automation which I'm not able to force to zero-indent :P